packages/mobile/src/common/enums/ui/button-type.enum.ts

```diff
@@ -0,0 +1,11 @@
+enum ButtonType {
+  PRIMARY = 'primary',
+  SECONDARY = 'secondary',
+}
+
+enum ButtonView {
```

Comment:
`View` might be a slightly confusing naming
```suggestion
enum ButtonAppearance {
```

Reply:
updated